### PR TITLE
Parameterize more browser tests. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: test/test_browser.py [hunk 1]
```diff
@@ -4256,21 +4256,26 @@ def test_pthread_custom_pthread_main_url(self):
...
   # Test that sbrk() operates properly in multithreaded conditions
   @requires_threads
-  def test_pthread_sbrk(self):
-    for aborting_malloc in [0, 1]:
-      print('aborting malloc=' + str(aborting_malloc))
-      # With aborting malloc = 1, test allocating memory in threads
-      # With aborting malloc = 0, allocate so much memory in threads that some of the allocations fail.
-      self.btest_exit('pthread/test_pthread_sbrk.cpp', args=['-O3', '-pthread', '-sPTHREAD_POOL_SIZE=8', '-sABORTING_MALLOC=' + str(aborting_malloc), '-DABORTING_MALLOC=' + str(aborting_malloc), '-sINITIAL_MEMORY=128MB'])
+  @parameterized({
+    '': (['-DABORTING_MALLOC=0', '-sABORTING_MALLOC=0'],),
```
Comment (kripken) [inline test/test_browser.py:4269]:
```suggestion
    '': (['-DABORTING_MALLOC=0'],),
```

**Review by kripken — COMMENTED** [review]
lgtm % comments

File: test/test_browser.py (hunk 1 above)
Comment (sbc100) [reply test/test_browser.py:4269]: I think this flag is needed to actually disable the ABORTING_MALLOC setting.

File: test/test_browser.py [hunk 2]
```diff
@@ -4605,9 +4618,11 @@ def test_small_js_flags(self):
...
       self.assertLess(abs(size - 4800), 100)
 
-  # Tests that it is possible to initialize and render WebGL content in a pthread by using OffscreenCanvas.
-  # -DTEST_CHAINED_WEBGL_CONTEXT_PASSING: Tests that it is possible to transfer WebGL canvas in a chain from main thread -> thread 1 -> thread 2 and then init and render WebGL content there.
-  @no_chrome('see https://crbug.com/961765')
+  # Tests that it is possible to initialize and render WebGL content in a
+  # pthread by using OffscreenCanvas.  -DTEST_CHAINED_WEBGL_CONTEXT_PASSING:
+  # Tests that it is possible to transfer WebGL canvas in a chain from main
+  # thread -> thread 1 -> thread 2 and then init and render WebGL content there.
+  # @no_chrome('see https://crbug.com/961765')
```
[the remark this answers is not in the document]
Comment (sbc100) [reply test/test_browser.py:4625]: Oops.  Done

File: test/test_browser.py (hunk 1 above)
Comment (kripken) [reply test/test_browser.py:4269]: Oh, sorry, my brain didn't realize one was `-s` and one `-D`...

**Review by kripken — APPROVED** [review]

**Pull request merged**